README.md

```diff
@@ -213,11 +213,7 @@ var pubsub = gcloud.pubsub({
   keyFilename: '/path/to/keyfile.json'
 });
 
-// Create a new topic.
-pubsub.createTopic('my-new-topic', function(err, topic) {});
-
-// Reference an existing topic.
-var topic = pubsub.topic('my-existing-topic');
+var topic = pubsub.topic('my-topic');
```

Comment:
Comment here about what this is doing... e.g. referencing a potentially non-existent topic? Auto-create is true by default... etc?